openlibrary/i18n/messages_cs.po

```diff
@@ -1,2179 +0,0 @@
-# Translation for Open Library.
```

Comment:
There are some good changes in this PR, but it has become stale though :( 

I don't think deleting this cs.po file is the right thing to do. There's not too much in it, but there is a way to auto-refresh the .po from the latest .pot which should be the process when changes are made to the .pot.

Many, but not all, of the menu strings have been translated into Czech, and will still be current: https://openlibrary.org/?lang=cs

Reply:
I don't understand the comment. It's only being deleted because it's a duplicate in the wrong place. The correct one is here: 
https://github.com/internetarchive/openlibrary/blob/09729ab073f68ca276f3fb8fb53dc77963cc3819/openlibrary/i18n/cs/messages.po

The PR will, of course, get rebased when it's finalized and no longer WIP. I can push a rebased version now, but the reason for the WIP tag is that I don't consider it ready to review yet.

Reply:
OK, it's been rebased, if that helps, but it's still WIP and not ready for review.

Reply:
@hornc Does the explanation about the cs.po file make sense? If so, can you mark this as resolved?

Reply:
@hornc Ping. I'd really like to get this merged in the next day or two so that it can't get some serious testing on dev.

Reply:
@tfmorris yes, makes sense -- looking back at the code it looks like this file was copied rather than moved to the correct location. I see now this PR deletes a duplicate .po file and leaves the correct one.

I hadn't realised there were 2 copies - It surprised me.